src/lib/yang/value.lua

```diff
@@ -95,8 +95,21 @@ function types.identityref.tostring(val)
    return val
 end
 
-types['instance-identifier'] = unimplemented('instance-identifier')
-types.leafref = unimplemented('leafref')
+types['instance-identifier'] = {}
+types['instance-identifier'].parse = function (str, what)
+   return assert(str, 'missing value for '..what)
```

Comment:
I guess this is temporary.  Do you plan to implement this later?  Needs a comment.